blockchain/blocks/src/header.rs

```diff
@@ -25,6 +26,36 @@ use std::time::{SystemTime, UNIX_EPOCH};
 const SHA_256_BITS: usize = 256;
 const BLOCKS_PER_EPOCH: u64 = 5;
 
+/// Proof of spacetime proof
+#[derive(Eq, PartialEq, Debug, Clone, Default)]
+pub struct PostProof {
+    registered_proof: u64,
+    proof_bytes: Vec<u8>,
+}
```

Comment:
This type exists in `vm/src/sector/post.rs` Should probably use that especially for registered_proof consistency

Reply:
Oh shit, nice find. I didn't see that...